client/src/components/BuildDetails.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+class BuildDetails extends React.Component {
+  renderBuild = build => {
+    return (
+      <React.Fragment key={build.buildid}>
+        <div className='col-sm-3'>
+          <a
+            target='_blank'
+            rel='noreferrer noopener'
+            href={`https://build.reactos.org/#builders/${
+              build.builderid
+            }/builds/${build.number}`}
+          >
+            {this.props.builders[build.builderid - 1].name}
+          </a>
+        </div>
+        <div className='col-sm-3'>
+          {build.state_string}
+          {build.state_string === 'build successful' ? (
```

Comment:
I think we should define some dictionary with build states, and convert this state_string to a value from that dictionary in reducer